### PR TITLE
PR-2387: Test to validate datetime handling in loadData.

### PR DESCRIPTION
**Pull request by XDelphiGrl**
[Liquibase PR](https://github.com/liquibase/liquibase/pull/2387)
[Liquibase Issue](https://github.com/liquibase/liquibase/issues/2383)

**Items of Note**

* The SQL in this test uses the `now()` function native to each database.
* This means that each time you run the test, the expected SQL will be different because `now` is never the same twice.
* I had to change the JSONCompareMode to LENIAT in JSONUtils.groovy; without this change, the test cannot pass.
  * If this is an unacceptable change, let me know. I can either close the pull request or revise according to your feedback.

This cannot be merged until the next release of Liquibase!

### CODE REVIEW

File: src/main/groovy/liquibase/harness/util/JSONUtils.groovy [hunk 1]
```diff
@@ -70,7 +74,10 @@ class JSONUtils {
             for (int j = 0; j < jsonArrayToCompare.length(); j++) {
                 def jsonObjectRight = new JSONObject(jsonArray.get(i).toString())
                 def jsonObjectLeft = new JSONObject(jsonArrayToCompare.get(j).toString())
-                def result = JSONCompare.compareJSON(jsonObjectLeft, jsonObjectRight, JSONCompareMode.NON_EXTENSIBLE)
```
Comment (KushnirykOleh) [inline src/main/groovy/liquibase/harness/util/JSONUtils.groovy:73]: @XDelphiGrl I added CustomComparator to be able to leave NON_EXTENSIBLE but match by values by regexp, that should make our checks more precise

**Review by KushnirykOleh — APPROVED** [review]

**Comment by KushnirykOleh** [comment]
@AlexanderSashchenko plz review as well to make sure we keep checks like you designed

File: src/main/groovy/liquibase/harness/util/JSONUtils.groovy (hunk 1 above)
Comment (XDelphiGrl) [reply src/main/groovy/liquibase/harness/util/JSONUtils.groovy:73]: @KushnirykOleh, thank you!!! I knew there had to be a better way to handle this particular case. Thank you for making my overly-broad code change much more precise!

**Review by AlexanderSashchenko — APPROVED** [review]
LGFM

**Pull request merged**